CHANGELOG.md

```diff
@@ -19,13 +19,124 @@
 
 # Table of Contents
 
+- [1.0.0](#100)
 - [0.6.0](#060)
 - [0.5.0](#050)
 - [0.4.0](#040)
 - [0.3.0](#030)
 - [0.2.0](#020)
 - [0.1.0](#010)
 
+# 1.0.0
+
+Welcome to the 1.0.0 release of apisix-ingress-controller!
+
+This is the first **GA** release.
+
+## Highlights
+
+### New Features
+
+* **Support blocklist-source-range annotation for Ingress source** [#446](https://github.com/apache/apisix-ingress-controller/pull/446)
+* **Add ApisixConsumer CRD** [#462](https://github.com/apache/apisix-ingress-controller/pull/462)
+* **Support rewrite annotation for Ingress source** [#480](https://github.com/apache/apisix-ingress-controller/pull/480)
+* **Support http-to-https redirect annotation for Ingress source** [#484](https://github.com/apache/apisix-ingress-controller/pull/484)
+* **Add health check to apisix-admin and make the leader election recyclable** [499](https://github.com/apache/apisix-ingress-controller/pull/499)
+* **Support mTLS for ApisixTls** [#492](https://github.com/apache/apisix-ingress-controller/pull/492)
+* **Support authentication for ApisixRoute** [#528](https://github.com/apache/apisix-ingress-controller/pull/528)
+
+Please try out the release binaries and report any issues at
+https://github.com/apache/apisix-ingress-controller/issues.
+
+### Contributors
+
+* Alex Zhang
+* Sarasa Kisaragi
+* Jintao Zhang
+* kv
+* Shuyang Wu
+* Daming
+* Fang
+* Ayush das
+* Donghui0
+* Shivani chauhan
+* Yuelin Zheng
+* guoqqqi
+* 罗泽轩
+
+### Changes
+<details><summary>58 commits</summary>
```

Comment:
commits are too much. OR just show a link here ?

Reply:
The commits are added to allow people to clearly understand what happened. I refer to some other big projects
They all included commits.